developer-docs/bolt_server.md

```diff
@@ -0,0 +1,39 @@
+# Bolt Server 
+
+## Overview
+Bolt server provides a service to run bolt tasks over SSH and WinRM. This service is exposed via an API as described in this document. 
+
+## Configuration
+Bolt server can be configured by defining content in HOCON format at one of the following expected configuration file path locations.
+**Global Config**: ` /etc/puppetlabs/bolt-server/conf.d/bolt-server.conf`
+**Local Config**: `~/.puppetlabs/bolt-server.conf`
+**Options**
+`port [optional int]`: The port the bolt server will run on (default 8144)
+`ssl-cert [required string]`: Path to the cert file
```

Comment:
I'd add a comment that in PE this defaults to `/etc/puppetlabs/bolt-server/$HOSTNAME.cert.pem`